scripts/normalize-by-median.py

```diff
@@ -126,6 +128,22 @@ def handle_error(error, output_name, input_name, fail_save, htable):
         print >> sys.stderr, '** ERROR: problem removing corrupt filtered file'
 
 
+@contextmanager
+def FailSafe(ifile, ofile, save_on_fail, ht, corrupted, total, dicarded, jedi):
```

Comment:
It was to fit in 80 chars. I can change it, but I'll have to throw in a line break.